src/util/misc/planeChange.ts

```diff
@@ -3,12 +3,23 @@ import { invertTransform, multiplyTransformMatrices, transformPoint } from "./ma
 import { iMatrix } from "../../constants";
 import type { IPoint, Point } from "../../point.class";
 import { applyTransformToObject } from './objectTransforms';
+import { TObject } from "../../__types__";
 
 export const enum ObjectRelation {
   sibling = 'sibling',
   child = 'child',
 }
 
+/**
+ * We are actually looking for the transformation from the destination plane to the source plane (change of basis matrix)\
+ * The object will exist on the destination plane and we want it to seem unchanged by it so we invert the destination matrix (`to`) and then apply the source matrix (`from`)
+ * @param [from]
+ * @param [to] 
+ * @returns 
+ */
+export const calcPlaneChangeMatrix = (from?: TMat2D | null, to?: TMat2D | null) =>
+  multiplyTransformMatrices(invertTransform(to || iMatrix), from || iMatrix);
```

Comment:
please let's keep iMatrix as the defaul value for arguments.
Those are my following reasons:
- default valules are a comfortable es6 feature
- i really can't understnad why we have to add `null` to typings for things that can just be undefined when omitted and be optional arguments